WalletWasabi/Affiliation/IAffiliationManager.cs

```diff
@@ -0,0 +1,8 @@
+using WalletWasabi.Affiliation.Models;
+
+namespace WalletWasabi.Affiliation;
+
+public interface IAffiliationManager
```

Comment:
It seems this interface is not necessary. Please remove it.

Reply:
The interface is used in tests. See [this](https://github.com/trezor/WalletWasabi/blob/ddba48534fbc090f20e577ac82e42abc0ab7cfdc/WalletWasabi.Tests/UnitTests/WabiSabi/Integration/WabiSabiApiApplicationFactory.cs#L68).

Reply:
That's because the AffiliationManager doesn't receive its dependencies and then you have to replace it completely with a fake/mock instance. 

In UTs we have to break dependencies with the external world, in this case we have to break dependency with the network so, you could receive a factory method that creates IHttpClient.

Reply:
> That's because the AffiliationManager doesn't receive its dependencies and then you have to replace it completely with a fake/mock instance.

The question is whether you want to use the real `AffiliationManager` in tests if it is not used by anyone.

> In UTs we have to break dependencies with the external world, in this case we have to break dependency with the network so, you could receive a factory method that creates IHttpClient.

Sorry, I don't understand what you mean. I feel this is closely related to https://github.com/zkSNACKs/WalletWasabi/pull/9769#discussion_r1048694574. So I suggest to solve that issue first.

Reply:
> The question is whether you want to use the real AffiliationManager in tests 

Yes. That's the idea.

Reply:
Is this what you expected? https://github.com/zkSNACKs/WalletWasabi/pull/9769/commits/719c21d2c87c6eafe7f0096245b221c85b8d66f6

Reply:
yes.